cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -241,7 +241,7 @@ match_types! {
 	};
 	pub type ParentOrSiblings: impl Contains<MultiLocation> = {
 		MultiLocation { parents: 1, interior: Here } |
-		MultiLocation { parents: 1, interior: X1(_) }
+		MultiLocation { parents: 1, interior: X1(Parachain(_)) }
 	};
```

Comment:
I would also rename the type to smth like `ParentOrSiblingParachains` or even `ParentRelayOrSiblingParachains` to make it clear this is to be used  by parachains for same-relay-ecosystem filters

Reply:
Agreed, I also moved this to parachains-common to know it's being used for our use-case of relay and parachains